HTTP_API_Guide.md

```diff
@@ -0,0 +1,256 @@
+HTTP API
+========
+
+The [Bloomberg Open API] provides access to market data. The
+[Bloomberg HTTP API] makes the [Open API] available via HTTP: clients POST
+commands to a HTTP server, which will respond with the relevant data.
+
+The [HTTP API] is currently in development, and does not yet provide access
+to all features of the [Open API]. At this time it supports the Reference Data
+Service (i.e. //blp/refdata) and the API Field Service (//blp/apiflds).
+
+This document provides an overview of the [HTTP API]: the URLs to use and the
+formatting for the HTTP requests and responses. We assume, for the purposes of
+this doc, that the server at http://blpapihost.example.com/ is running an
+instance of this API.
+
+This document also includes basic information about some of the underlying
+[Open API] operations. For more details on the [Open API], refer to the
+[BLPAPI Developer's Guide]. This document will often refer to specific sections
+of the [Developer's Guide] where appropriate.
+
+[Bloomberg Open API]: http://bloomberglabs.com/api
+[Open API]: http://bloomberglabs.com/api
+[Bloomberg HTTP API]: http://github.com/bloomberg/blpapi-http
+[HTTP API]: http://github.com/bloomberg/blpapi-http
+[BLPAPI Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
+[Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
+
+
+Requesting data
+---------------
+
+To request data, a client makes a HTTP POST request on a URL of the following
+form:
+
+```
+https://blpapihost.example.com/request/blp/refdata/<operation>
+```
+
+Let's examine each segment of the path:
+
+* `/request` -- The Open API paradigm we're using, namely Request/Response.
+* `/blp/refdata` -- The service on which we're making the request. Currently
+`/blp/refdata` and `/blp/apiflds` are supported; more will be added in future
+versions.
+* `<operation>` -- The operation for this request. For `/blp/refdata`,
+the operations are listed in the [Developer's Guide] in section A.2.1 (and
```

Comment:
Switch sections and reference to the guide.  e.g., "the operations are listed in section A.2.1 of the [Developer's Guide]."